projects/portal/src/app/views/dialogs/wallets/ununifi/ununifi-import-wallet-with-mnemonic-form-dialog/ununifi-import-wallet-with-mnemonic-form-dialog.component.ts

```diff
@@ -66,7 +67,7 @@ export class UnunifiImportWalletWithMnemonicFormDialogComponent implements OnIni
             return {
               id,
               type: WalletType.ununifi,
-              mnemonic: mnemonicWithNoWhitespace,
+              mnemonic: mnemonic,
```

Comment:
I think this is strange.
Why do you think so?
I guess you can use mnemonicWithNoWhitespace here, but, is there any special reason?
I want to confirm it.

Reply:
I think `privateWallet$` should have a raw mnemonic string. 

Because when the mnemonics are changed (ex. trim), the cursor goes to the end in the form.
I think it difficult to operate. espesialy when edit middle mnemonic (not last).

I think the string should be arranged (trimed) only when the `privateWallet$` are passed to others , or  calculate the private key.